phpari.php

```diff
@@ -75,18 +75,53 @@ class phpari
 
 	/**
 	 * @param null $stasisApplication
-	 * @param string $configFile
+	 * @param <mixed> $config
 	 *
 	 * Returns an array containing 5 objects: WebSocket, Pest, EventLoopFactory, Logger, StasisEventHandler
 	 *
 	 */
-	public function __construct($stasisApplication = NULL, $configFile = "phpari.ini")
+	public function __construct($stasisApplication = NULL, $config = "phpari.ini")
 	{
 		try {
-
-			/* Get our configuration */
-			$this->configuration = (object)parse_ini_file($configFile, TRUE);
-
+			/* if config is an array pull configuration from the array */
+			if(is_array($config)) {
```

Comment:
Personally speaking, I normally prefer to put the shorter options first. Thus, I would start the code with:
```
if (!is_array($config)) {
  // File Handling
} else {
  // Array handling
}
```
The main reason for that is to make the code a little more readable and accessible.